test/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.FunctionalTests/ApplicationConsumingPrecompiledViews.cs

```diff
@@ -22,85 +16,26 @@ public ApplicationConsumingPrecompiledViews(ApplicationConsumingPrecompiledViews
 
         public ApplicationTestFixture Fixture { get; }
```

Comment:
Pretty sure xunit disposes `IClassFixture` if they implement `IDisposable`. I don't think this is necessary.